From 6cb5ffe48d6a980386f904ad1fc512dac54c7b47 Mon Sep 17 00:00:00 2001
From: ripee <ripee.dev@gmail.com>
Date: Sat, 18 May 2019 23:06:58 +0200
Subject: [PATCH] libdisplaymodule --> libexynosdisplaymodule

---
 libhwc/Android.mk | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/hardware/samsung_slsi-cm/exynos/libhwc/Android.mk b/hardware/samsung_slsi-cm/exynos/libhwc/Android.mk
index c231e8f..0fd6abc 100644
--- a/hardware/samsung_slsi-cm/exynos/libhwc/Android.mk
+++ b/hardware/samsung_slsi-cm/exynos/libhwc/Android.mk
@@ -21,7 +21,7 @@ LOCAL_MODULE_RELATIVE_PATH := hw
 LOCAL_SHARED_LIBRARIES := liblog libcutils libEGL libGLESv1_CM libhardware \
 	libhardware_legacy libion_exynos libutils libsync \
 	libexynosv4l2 libMcClient libexynosutils libhwcutils libexynosdisplay libhdmi \
-	libdisplaymodule libhdmimodule libmpp libbase libnativewindow
+	libexynosdisplaymodule libhdmimodule libmpp libbase libnativewindow
 
 LOCAL_STATIC_LIBRARIES := libarect
 
@@ -106,7 +106,7 @@ LOCAL_C_INCLUDES += \
 	$(TOP)/hardware/samsung_slsi-cm/exynos/libcec \
 	$(TOP)/hardware/samsung_slsi-cm/$(TARGET_SOC)/include \
 	$(TOP)/hardware/samsung_slsi-cm/$(TARGET_SOC)/libhwcmodule \
-	$(TOP)/hardware/samsung_slsi-cm/$(TARGET_SOC)/libdisplaymodule \
+	$(TOP)/hardware/samsung_slsi-cm/$(TARGET_SOC)/libexynosdisplaymodule \
 	$(TOP)/hardware/samsung_slsi-cm/$(TARGET_SOC)/libhdmimodule \
 	$(TOP)/hardware/samsung_slsi-cm/$(TARGET_SOC)/libhwcutilsmodule \
 	$(TOP)/hardware/samsung_slsi-cm/exynos/libmpp \
-- 
2.17.1

